src/NServiceBus.AcceptanceTesting/FodyWeavers.xml

```diff
@@ -0,0 +1,3 @@
+﻿<Weavers xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="FodyWeavers.xsd">
```

Comment:
We should instead remove Fody.Janitor and Fody and remove the [SkipWeaving] directive from AcceptanceTestingOutboxTransaction